memo/managers.py

```diff
@@ -73,3 +74,63 @@ def get_card_amount_in_deck(self, deck):
 class DeckManager(models.Manager):
     def get_decks_by_owner(self, user):
         return self.filter(owner=user)
+
+
+class CardAnswerHistoryManager(models.Manager):
+    def get_consecutive_days(self, user):
+        from .models import Card
+
+        max_daily_date_times = (
+            self.filter(owner=user)
+            .extra(select={"day": "date( created_at )"})
+            .values("day")
+            .annotate(latest_datetime=Max("created_at"))
+        )
+
+        objects = (
+            self.filter(
+                created_at__in=[
+                    entry["latest_datetime"] for entry in max_daily_date_times
+                ]
+            )
+            .values("card", "created_at")
+            .order_by("-created_at")
+        )
```

Comment:
Just removing the `values` method and replacing the `entry["latest_datetime"]` for the `entry.created_at`
```suggestion
        objects = self.filter(
            created_at__in=[entry.created_at for entry in max_daily_date_times]
        ).order_by("-created_at")
```

Reply:
@r-lelis And resuming the text from my last reply, the `latest_datetime` is needed to iterate through dict because I need to run: 
```
SELECT card_id, created_at 
FROM memo_cardanswerhistory
WHERE created_at 
IN ('2022-08-04 04:41:59.605 -0300', '2022-08-02 05:30:59.605 -0300', '2022-08-01 16:32:59.605 -0300)
```
and it will give me:

| CARD_ID      | CREATED_AT |
|:--------|:---------|
| 4  | 2022-08-04 04:41:59.605 -0300  |
| 3 | 2022-08-02 05:30:59.605 -0300  |
| 1   | 2022-08-01 16:32:59.605 -0300   |

Getting the `card_id` and `created_at` that'll need to return in api response.

Reply:
Same as the other answer, you can leave it as it is =]